File: mine/src/main/java/com/klcw/app/mine/MineApplication.java
package com.klcw.app.mine;

import android.app.Application;

import com.klcw.app.lib.network.NetworkConfig;
/**
 * @author kk
 * @datetime: 2018/10/24
 * @desc:
 */
public class MineApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        NetworkConfig.setup(this, "prd");
    }
}

File: mine/src/main/java/com/kk/app/mine/MineApplication.kt
package com.kk.app.mine

import android.app.Application
import com.kk.app.lib.network.NetworkConfig

/**
 * @author kk
 * @datetime: 2018/10/24
 * @desc:
 */
class MineApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        NetworkConfig.setup(this, "prd")
    }
}